pkg/cloudprovider/providers/gce/gce_disks.go

```diff
@@ -535,14 +535,10 @@ func (gce *GCECloud) AttachDisk(diskName string, nodeName types.NodeName, readOn
 	if regional && utilfeature.DefaultFeatureGate.Enabled(features.GCERegionalPersistentDisk) {
 		disk, err = gce.getRegionalDiskByName(diskName)
```

Comment:
what is regional is true, but feature is not enabled, or this case is not possible at all.

Reply:
In that case we fall back to the behavior for a regular disk, as if the regional PD feature doesn't exist.

`regional` is set if the volume is a PV spec and its zone label has the regional PD format. If the feature gate is disabled, when a user creates a PV with regional PD zone label, the PV is rejected by an admission controller, so usually we don't hit this scenario.